packages/victory-cursor-container/src/index.d.ts

```diff
@@ -4,7 +4,7 @@ import { VictoryContainerProps, CursorData } from "victory-core";
 export interface VictoryCursorContainerProps extends VictoryContainerProps {
   cursorComponent?: React.ReactElement;
   cursorDimension?: "x" | "y";
-  cursorLabel?: (point: CursorData) => void;
+  cursorLabel?: (point: CursorData) => number;
```

Comment:
This is incorrect. `cursorLabel` should be able to return a string, too.

Reply:
Shall I do string | number, or is there a situation where void should be accepted too?

Reply:
Yeah, folks sometimes return null to render no label for a given data point. Let's leave this one as flexible as possible.